node_package/src/serverRenderReactComponent.ts

```diff
@@ -110,8 +110,12 @@ as a renderFunction and not a simple React Function Component.`);
       let promiseResult;
 
       try {
+        let awaitedResult = await renderResult;
+        if(typeof awaitedResult !== 'string' && typeof awaitedResult !== 'object'){
```

Comment:
typeof a function is surely not object, correct/

Reply:
I was thinking more of an object like `{ componentHtml: ..., some other react-helmet key-pairs }`